atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/RowResult.java

```diff
@@ -68,6 +68,10 @@ public byte[] getRowName() {
         return row.clone();
     }
 
+    public long getRowNameSize() {
```

Comment:
Added only because `getRowName` copies data (& also because the parameter type `T` in row result won't necessarily implement `Measurable` so we can't implement this in-class)